src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx

```diff
@@ -204,15 +204,25 @@ const UnitsTab: FC<UnitsTabProps> = ({ trackingData, formattedData }) => {
 
   return (
     <Box>
-      <Box $maxWidth={1280} $mh={"auto"} $ph={[0, 18]} $width={"100%"}>
-        <OakHeading
-          tag="h2"
-          $mb="space-between-m"
-          $ml={["space-between-s", "space-between-none"]}
-          $font={["heading-5", "heading-4"]}
-        >
-          Unit sequence
-        </OakHeading>
+      <Box
+        id="curriculum-units"
+        aria-labelledby="curriculum-unit-sequence-heading"
```

Comment:
See previous comment re: use of `aria-labelledby`

Reply:
As above,  I've added the role attribute